specs/TLA+/LeaderFollower/MCAbstractVeniceLeaderFollower.tla

```diff
@@ -0,0 +1,12 @@
+----------------------- MODULE MCAbstractVeniceLeaderFollower -----------------
+EXTENDS AbstractVeniceLeaderFollower
+
+CONSTANT MAX_WRITES
+
+AllWritesTransmitted ==
+    Len(realTimeTopic) <= MAX_WRITES
```

Comment:
So `AllWritesTransmitted` should mean `realTimeTopic` is empty, right?

Reply:
We should probably change the name of this function.  This is the Constraint for the spec, and this tells the model checker to not explore states where this is is not true.  So it's not AllWritesTransmitted, it's NotAllWritesTransmitted.  We can probably also get rid of the 'Terminating' function now that we don't count version pushes just to simplify things and make NotAllWritesTransmitted the Constraint.

Reply:
Nitpicking here: A *function* in TLA+ is what we typically call a map or dict in programming, i.e. `[ i \in 1..3 |-> i * 2]`.  `AllWritesTransmitted` is a *state predicate* in TLA+ parlance, which is true or false of a state.  If a state predicate appears as a *state constraint* in TLC, it does what @ZacAttack says above. :-)